subprojects/android-lib/snackbar-proxy/src/main/kotlin/com/avito/android/snackbar/proxy/SnackbarProxy.kt

```diff
@@ -0,0 +1,20 @@
+package com.avito.android.snackbar.proxy
+
+import androidx.annotation.VisibleForTesting
+import com.google.android.material.snackbar.Snackbar
+
+// VisibleForTesting doesn't work in Kotlin https://issuetracker.google.com/issues/140642032
+// Fixed in AS 4.0
```

Comment:
It's a stable release and it works with AGP 3+. It's no use to comment old issues.

Reply:
I faced that VisibleForTesting didn't work correctly

Reply:
Do you use AS 3+?